.deepsource.toml

```diff
@@ -0,0 +1,11 @@
+version = 1
```

Comment:
I think we should exclude this file.

Reply:
@xhebox should I remove the `.toml` file ?

Reply:
I personally prefer to remove. And I think PR will get merged easier if it is removed. This file may be controversial.

@andylokandy  I've shared deepsource to devs. But there is no discussion yet.

Reply:
@xhebox removed `.toml` file.